### ISSUE TITLE
Popover header is not changing after changing the theme to subtle in iOS

### ISSUE THREAD

**Issue by janahintal** [infor-design/enterprise#3032]
**Describe the bug**
Popover header is not changing after changing the theme to subtle in iOS

**To Reproduce**
1. Open iOS device and go to http://master-enterprise.demo.design.infor.com/components/popover/example-header-color.html
2. Change the Theme to Vibrant 
3. Click the popover button
4. Change the Theme back to Subtle
5. Click the popover button

**Expected behavior**
Popover header and the Body header should be in Subtle 

**Version**
Beta 4.23
**Screenshots**
![image](https://user-images.githubusercontent.com/45757016/67567029-bb6bce80-f75b-11e9-9b15-d7c3407890ce.png)

**Platform**
iOS Devices - Tested in XR ios 13

**Additional context**
N/a

**Comment by tmcconechy** [comment]
I agree that should probably change to the personalization color. Need to be aware of one thing when making this change that the header goes with the color on calendar when clicking events.

http://master-enterprise.demo.design.infor.com/components/calendar/example-index.html
